Planner/app/src/main/java/rdatu/android/cyscorpions/com/projectplanner/controller/TaskManager.java

```diff
@@ -0,0 +1,114 @@
+package rdatu.android.cyscorpions.com.projectplanner.controller;
+
+import android.content.Context;
+import android.util.Log;
+
+import java.util.ArrayList;
+
+import rdatu.android.cyscorpions.com.projectplanner.model.Tasks;
+import rdatu.android.cyscorpions.com.projectplanner.model.TasksDatabaseHelper;
+import rdatu.android.cyscorpions.com.projectplanner.model.TasksDatabaseHelper.TaskCursor;
+
+/**
+ * Created by rayeldatu on 7/28/15.
+ */
+public class TaskManager {
+
+    private static TaskManager sTaskManager;
+    private Context mAppContext;
+    private TasksDatabaseHelper mHelper;
+    private ArrayList<Tasks> mTasks;
+    private Tasks mLoadedTask;
+
+    private TaskManager(Context context, boolean all) {
```

Comment:
What does the variable "all" stand for? Please make your variable names more descriptive, so it's more clear what it is representing :bow: